src/reporter/tests/test_instanceId.py

```diff
@@ -0,0 +1,57 @@
+from exceptions.exceptions import NGSIUsageError, InvalidParameterValue
+from conftest import QL_URL
+from datetime import datetime
+from reporter.tests.utils import insert_test_data, delete_test_data
+from utils.tests.common import assert_equal_time_index_arrays
+from translators.factory import translator_for
+import pytest
+import requests
+import dateutil.parser
+
+entity_type = "Room"
+entity_id = "Room1"
+n_days = 6
+services = ['t1', 't2']
+
+
+def query_url(etype=entity_type, values=False):
+    url = "{qlUrl}/types/{entityType}"
+    if values:
+        url += '/value'
+    return url.format(
+        qlUrl=QL_URL,
+        entityType=etype
+    )
+
+
+@pytest.fixture(scope='module')
+def reporter_dataset():
+    for service in services:
+        insert_test_data(service, [entity_type], n_entities=3,
+                         entity_id=entity_id, index_size=n_days)
+    yield
+    for service in services:
+        delete_test_data(service, [entity_type])
+
+@pytest.mark.parametrize("service", services)
+def test_instanceId(service, reporter_dataset):
+    import pdb
```

Comment:
this is why the test fails on circleci, remove pdb here and below

Reply:
Thanks a lot for pointing this out. I forgot to remove pdb.